content/networks-of-brokers.html

```diff
@@ -584,10 +584,12 @@ <h3 id="example-configuration-using-networkconnector-properties">Example Configu
 <p><strong>N.B.</strong> You can only use <a href="wildcards">wildcards</a> in the <code class="highlighter-rouge">excludedDestinations</code> and <code class="highlighter-rouge">dynamicallyIncludedDestinations</code> properties.<br />
 <strong>N.B.</strong> <strong>Do not</strong> change the name of the bridge or the name of the Broker if you are using durable topic subscribers across the network. Internally ActiveMQ uses the network name and broker name to build a unique but repeatable durable subscriber name for the network.</p>
 
-<h3 id="stuck-messages-version-56">Stuck Messages (version 5.6)</h3>
+<h3 id="stuck-messages">Stuck Messages</h3>
 
-<p>By default, it is not permissible for a message to be replayed back to the broker from which it came. This ensures that messages do not loop when duplex or by directional network connectors are configured. Occasionally it is desirable to allow replay for queues. Consider a scenario where a bidirectional bridge exists between a broker pair. Producers and Consumers get to randomly choose a broker using the failover transport. If one broker is restarted for maintenance, messages accumulated on that broker, that crossed the network bridge, will not be available to consumers till they reconnect to the broker. One solution to this problem is to force a client reconnect using <a href="http://activemq.apache.orgUsing ActiveMQ/Configuring Transports/ActiveMQ Connection URIs/failover-transport-reference.md#FailoverTransportReference-BrokersideOptionsforFailover">rebalanceClusterClients</a>. Another, is to allow replay of messages back to the origin as there is no local consumer on that broker.<br />
-There is a destination policy that allows this behavior for queues by configuring a <code class="highlighter-rouge">conditionalNetworkBridgeFilterFactory</code> with <code class="highlighter-rouge">replayWhenNoConsumers=true</code>. The <code class="highlighter-rouge">conditionalNetworkBridgeFilterFactory</code> provides an optional <code class="highlighter-rouge">replayDelay</code> based on the broker-in time.</p>
+<p>There might have multiples cause for Stuck Message, paragrapĥs below aim to give advice setting that may help to fix this issue.</p>
```

Comment:
sections is more accurate than paragraphs.

Reply:
I meant to use `sections` instead of `paragraphs` (especially there's a weird type).

Reply:
Ok I replace paragraphs by sections